storage/blockdevice/COMPONENT_OSPIF/include/OSPIF/MX25LM51245G_config.h

```diff
@@ -42,4 +42,17 @@ uint8_t _sfdp_basic_param_table[64] = {0x30, 0xFF, 0xFB, 0xFF, 0xFF, 0xFF, 0xFF,
 uint8_t _sfdp_4_byte_inst_table[8] = {0x7F, 0xEF, 0xFF, 0xFF, 0x21, 0x5C, 0xDC, 0x14};
 #endif
 
+//#define MX_FLASH_SUPPORT_RWW 1
+
+#define MX25LM51245G_FLASH_SIZE                   0x4000000 /* 512 MBits => 64 MBytes */
+#define MX25LM51245G_BLOCK_SIZE                   0x10000   /* 1024 blocks of 64 KBytes */
+#define MX25LM51245G_SECTOR_SIZE                  0x1000    /* 16384 sectors of 4 kBytes */
+#define MX25LM51245G_PAGE_SIZE                    0x100     /* 262144 pages of 256 bytes */
+#define MX25LM51245G_CHUNK_SIZE                   0x10      /* fred: 16 bytes */
+
+#ifdef MX_FLASH_SUPPORT_RWW
+#define MX25LM51245G_BANK_SIZE                    0x01000000                    /* fred: 16 MBytes */
```

Comment:
Why you don't create a new file MX25LW51245G_config.h ?

Reply:
I have created a new file MX25LW51245G_config.h .